StickKnightsOnline-Client/main.cpp

```diff
@@ -1738,6 +1744,21 @@ void Button::handle_events(int ID)
 											current_sign = map[current_map]->Sign[i];
 											return;
 										}
+
+
+										//bool isColliding = blocked(Player[MyID].x, Player[MyID].y, map[current_map]->Sign[i].x, map[current_map]->Sign[i].y);
+
+
+										bool isColliding = ((Player[MyID].x == map[current_map]->Sign[i].x) && (Player[MyID].y == map[current_map]->Sign[i].y));
```

Comment:
this looks like a leftover failed attempt. It won't work this way so be sure to remove it